### ISSUE TITLE
Error in test_macros

### ISSUE THREAD

**Issue by costashatz** [resibots/limbo#58]
I am getting the following error in `test_macros`:

```
Running 1 test case...
../src/tests/test_macros.cpp(31): error in "test_macros": check (2 - 1) == 0 failed

*** 1 failure detected in test suite "macros"
```

Can you reproduce it?

**Comment by jbmouret** [comment]
No problem on my side (clang / osx),... but it fails on the cluster with g++ (4.9.2).

**Comment by fedeallocati** [comment]
Checked with the 3 compilers:
- ICC and clang passes
- GCC doesn't

I think we can just remove the line of the assertion, because it's checking that a call to the macro without parameters returns 0, and for us it's not necessary, since an Eigen Vector of dimension 0 would fail anyway

**Comment by jbmouret** [comment]
**ok** to remove the test.

**Comment by costashatz** [comment]
**OK**

**Comment by fedeallocati** [comment]
Fixed

**Comment by costashatz** [comment]
Fixed in  1defaba
